rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -909,14 +909,14 @@ func Qualify_SubProduct_FixedFloat: <"Qualifies a product as having the Sub Prod
         is_product boolean (1..1)
 
     set is_product:
-        (economicTerms -> payout -> interestRatePayout -> rateSpecification -> fixedRate count = 1
-                and economicTerms -> payout -> interestRatePayout -> rateSpecification -> floatingRate count = 1)
-            or (economicTerms -> payout -> interestRatePayout -> rateSpecification -> fixedRate count = 1
-                and economicTerms -> payout -> interestRatePayout -> rateSpecification -> inflationRate count = 1)
-            or ((economicTerms -> payout -> interestRatePayout -> rateSpecification -> fixedRate is absent
-                and economicTerms -> payout -> interestRatePayout -> principalPayment -> finalPayment only-element = True)
-                and (economicTerms -> payout -> interestRatePayout -> rateSpecification -> floatingRate count = 1
-                    or economicTerms -> payout -> interestRatePayout -> rateSpecification -> inflationRate count = 1))
+	 	(economicTerms -> payout -> interestRatePayout -> rateSpecification -> fixedRate count = 1
+	        and economicTerms -> payout -> interestRatePayout -> rateSpecification -> floatingRate count = 1)
+		or (economicTerms -> payout -> interestRatePayout -> rateSpecification -> fixedRate count = 1
+	        and economicTerms -> payout -> interestRatePayout -> rateSpecification -> inflationRate count = 1)
+		or ((economicTerms -> payout -> interestRatePayout -> rateSpecification -> floatingRate count = 1
+			    or economicTerms -> payout -> interestRatePayout -> rateSpecification -> inflationRate count = 1)
+            and (economicTerms -> payout -> interestRatePayout
+                filter item -> priceQuantity exists and item -> rateSpecification is absent then count = 1))
 
 func Qualify_SubProduct_FixedFixed: <"Qualifies a product as having the Sub Product classification Fixed Float">
```

Comment:
Nothing to do with your PR but the comment on line 921 says "Fixed Float" but this is qualifying "Fixed Fixed", any chance you could correct this while you are in here please?

Reply:
Yes, no problem. It is corrected now.